### ISSUE TITLE
Control.LogicalChildren.Add Doesn't set Parent property?

### ISSUE THREAD

**Issue by Kermalis** [AvaloniaUI/Avalonia#1993]
```cs
this.LogicalChildren.Add(child);
if (child.Parent == null)
    Console.WriteLine("Why is it null?"); // Gets run
```

Is there something else I need to do for the child's parent to be set? I have searched through source for hours and don't understand where else it could possibly be set.

**Comment by Kermalis** [comment]
It should be noted that the Parent property is properly set in the nightly builds, just not in 0.6.1

**Comment by grokys** [comment]
Closing as looks like this should have been fixed in 0.7.
